multiqc/modules/skewer/skewer.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from QualiMap """
+
+from __future__ import print_function
+
+import os
+from collections import OrderedDict
+import logging
+import re
+from multiqc import config, BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """ Skewer """
+
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='Skewer', anchor='skewer',
+                                            href="https://github.com/relipmoc/skewer.git",
+                                            info="is a tools to trim adapters off of reads")
+
+        self.skewer_data = dict()
+        for f in self.find_log_files(config.sp['skewer'], filehandles=True):
+            self.parse_skewer_log(f)
+
+        headers = OrderedDict()
+        headers['r_processed'] = {
+            'title': 'Read Pairs',
+            'description': 'Total number of read pairs processed',
+            'modify': lambda x: x / 1000000,
+            'min': 0,
+            'scale': 'RdYlGn',
+            'format': '{:.0f}'
```

Comment:
We can add the `shared_key` attribute here as it's a read count. Described in the footer of [this section of the docs](http://multiqc.info/docs/#step-4-adding-to-the-general-statistics-table)

See the equivalent read count section in the STAR module [here](https://github.com/ewels/MultiQC/blob/master/multiqc/modules/star/star.py#L116-L123)